lib/index.js

```diff
@@ -33,28 +36,54 @@ class Parser extends EventEmitter {
     this.license = pkg.license
   }
 
-  parse (sentence) {
-    if (typeof sentence === 'string' && sentence.trim().length > 0) {
-      this.emit('nmea0183', sentence.trim())
```

Comment:
This is something we want to keep: emitting the raw sentences. These are available within the server for plugins and for example over tcp.

Reply:
Would it be hard to emit the sentences from the same place where you call `parser.parse()`? 

I can re-introduce it but it seems pretty orthogonal to the library itself. There is absolutely no value added besides `.trim()`.